src/main/java/client/processor/npc/DueyProcessor.java

```diff
@@ -285,6 +285,12 @@ public static void dueySendItem(Client c, byte invTypeId, short itemPos, short a
         if (c.tryacquireClient()) {
             try {
                 int fee = Trade.getFee(sendMesos);
+                if (sendMessage.length() > 100) {
```

Comment:
Will throw `NullPointerException` for normal delivery where you don't provide a message.

Reply:
good point then this will fix that:
if (sendMessage.length() > 100) > if(sendMessage != null && sendMessage.length() > 100)